src/button/interrupt.rs

```diff
@@ -59,20 +62,19 @@ pub fn enable(external_interrupts: &EXTI, sysconfig: &SYSCFG, mode: TriggerMode)
 }
 
 fn configure_exti0(interrupt_mask: &IMR1) {
-    interrupt_mask.modify(|_, w| w.mr0().set_bit())
+    interrupt_mask.modify(|_, w| w.mr0().unmasked())
```

Comment:
Thinking out loud: are `configure_exti0` and `map_exti0_to_pa0` some typical Rust pattern? They are one-liners replaced with another one-liner, so I am wondering if it would be better for code readability to not have them.

Reply:
They’re not a Rust pattern. I was just naming things so I could understand the code without looking at the controller’s datasheet.

Reply:
I’ll be honest, I’ll need to look up the difference between `set_bit` and `unmasked`/`enabled`